CharacterClient/main.py

```
@@ -0,0 +1,66 @@
+    # Initialize the character client instance within the app state
+    initialize_character_client(
+        token=args.token,
+        Actor_id=args.Actor_id,
+        server_url=args.server_url,
+        client_port=args.client_port # Pass client_port for registration
+    )
+
+    # Start the heartbeat task (if character_client_instance is now initialized)
+    if hasattr(app.state, 'character_client_instance') and app.state.character_client_instance:
+        start_heartbeat_task(app.state.character_client_instance)
+        print(f"Client '{args.Actor_id}' API server starting on {args.client_host}:{args.client_port}")
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)